spec/support/simple_rest_client.rb

```diff
@@ -39,7 +36,7 @@ def version
       result = JSON.parse(response.body)
 
       { success: result['success'], payload: result }
-    rescue => e
+    rescue CannotFindErrorMessage => e
```

Comment:
This doesn't make sense to me unless `CannotFindErrorMessage` is some Ruby magic that neither I nor Google have heard of.

Using `rescue => e` sucks but it's a larger problem to tackle than resolving style violations.

Reply:
rubocop is suggesting this change?

Reply:
> rubocop is suggesting this change?

Impossible.

Reply:
Changed to `StandardError`.